### ISSUE TITLE
Not working properly

### ISSUE THREAD

**Issue by ageblade** [akiver/csgo-sharecode#12]
I tried to do a decode/encode test on the same Sharecode you posted in the Readme section,.
It always return `false`
```
const ShareCode = require('csgo-sharecode');

const match = ShareCode.decode('CSGO-GADqf-jjyJ8-cSP2r-smZRo-TO2xK');
const code = ShareCode.encode(match.matchId, match.reservationId, match.tvPort);
console.log(code === match);
```

**Comment by akiver** [comment]
You are comparing an object to a string which is not valid in JS.

I think what you are trying to do is: 

```js
const ShareCode = require('csgo-sharecode');

const expectedCode = 'CSGO-GADqf-jjyJ8-cSP2r-smZRo-TO2xK'
const match = ShareCode.decode(expectedCode);
const code = ShareCode.encode(match.matchId, match.reservationId, match.tvPort);
console.log(code === expectedCode);
```

**Comment by ageblade** [comment]
You are right, I will check it and close the issue if it's working properly

**Comment by ageblade** [comment]
OK so the thing is I'm trying to make sure your tool works.
I changed the testing code as you mentioned but still if I provide any sharecode from the latest games, it doesn't work properly. 

For example codes like:
CSGO-bPQEz-PrYTq-u5w8E-ZbUy7-ZeQ3A
CSGO-wBrm6-7fkM6-AzBC5-u6GmR-iHLHA

I fetched them from CS:GO watch section.

**Comment by akiver** [comment]
Fixed in 1.2.0, thanks for reporting!
